code/ZAS/Zone.dm

```diff
@@ -171,7 +173,7 @@ Class Procs:
 	M << name
 	for(var/g in air.gas)
 		M << "[gas_data.name[g]]: [air.gas[g]]"
-	M << "P: [air.return_pressure()] kPa V: [air.volume]L T: [air.temperature]�K ([air.temperature - T0C]�C)"
+	M << "P: [air.return_pressure()] kPa V: [air.volume]L T: [air.temperature]�K ([air.temperature - T0C]�C)"
```

Comment:
Okay maybe this is just the PC I'm on right now but this is showing as question mark error characters and this worries me